packages/rocketchat-ui-master/public/icons.svg

```diff
@@ -56,4 +56,6 @@
     <symbol viewBox="0 0 20 20" id="icon-video"><g stroke-width="1.5" stroke="currentColor" fill="none" fill-rule="evenodd"><path d="M3 8h10v8H3zm10 2.376l4.048-1.314c.526-.171.952.137.952.69v4.494c0 .552-.426.862-.952.69L13 13.624v-3.247z" /><path d="M5.5 5h5" stroke-linecap="square" /></g></symbol>
     <symbol viewBox="0 0 20 20" id="icon-volume"><path d="M3 6.345h1.967S6.027 4 9 4v12c-2.874 0-4.033-2.345-4.033-2.345H3v-7.31zm10.106 7.081A6.968 6.968 0 0 0 14 10a6.968 6.968 0 0 0-.894-3.426m2.489 8.809A10.95 10.95 0 0 0 17 10c0-1.956-.51-3.792-1.405-5.383" stroke="currentColor" stroke-width="1.5" fill="none" /></symbol>
     <symbol viewBox="0 0 20 20" id="icon-warning"><g stroke="currentColor" stroke-width="1.5" fill="none" fill-rule="evenodd"><path d="M10 3l8 14H2z" /><path d="M10 9v5" stroke-linecap="square" /></g></symbol>
+    <symbol id="icon-checkmark-circled" viewBox="0 0 1024 1024"><title>checkmark-circle</title><path class="path1" d="M486.4 1024c-129.922 0-252.067-50.594-343.936-142.464s-142.464-214.014-142.464-343.936c0-129.923 50.595-252.067 142.464-343.936s214.013-142.464 343.936-142.464c129.922 0 252.067 50.595 343.936 142.464s142.464 214.014 142.464 343.936-50.594 252.067-142.464 343.936c-91.869 91.87-214.014 142.464-343.936 142.464zM486.4 102.4c-239.97 0-435.2 195.23-435.2 435.2s195.23 435.2 435.2 435.2 435.2-195.23 435.2-435.2-195.23-435.2-435.2-435.2z"></path><path class="path2" d="M384 742.4c-6.552 0-13.102-2.499-18.102-7.499l-153.6-153.6c-9.997-9.997-9.997-26.206 0-36.203 9.998-9.997 26.206-9.997 36.205 0l135.498 135.498 340.299-340.298c9.997-9.997 26.206-9.997 36.203 0 9.998 9.998 9.998 26.206 0 36.205l-358.4 358.4c-5 4.998-11.55 7.498-18.102 7.498z"></path></symbol>
```

Comment:
Please convert your icons to 20x20 viewbox and remove the title and class. Thank you :)

Reply:
Sure. Will do it soon.